src/librustc_mir/borrow_check/borrow_set.rs

```diff
@@ -322,6 +330,10 @@ impl<'a, 'gcx, 'tcx> GatherBorrows<'a, 'gcx, 'tcx> {
             );
         };
 
+        // Consider the borrow not activated.
```

Comment:
I'd say "Consider the borrow not activated to start. When we find an activation, we'll update this field."

Reply:
Pushed improved comments.